kolibri/plugins/learn/assets/src/views/LearnExamReportViewer.vue

```diff
@@ -55,6 +56,16 @@
       ExamReport,
       ImmersivePage,
     },
+    setup() {
+      const { session } = useUser();
```

Comment:
Again, can destructure the specific properties here, rather than the session object.